src/App.js

```diff
@@ -117,6 +117,24 @@ class App extends Component {
     }
   }
 
+  saveEvent = async (user, event) => {
+    try {
+      const res = await this.props.auth0.getIdTokenClaims();
+      const jwt = res.__raw;
+      const config = {
+        method: 'post',
+        baseURL: `${process.env.REACT_APP_DB_URL}`,
```

Comment:
It looks like this should route through the server instead of the DB.

Reply:
We don't have it being saved to the server if that's what you are referring to. I think we would have to temp cache the stuff, but it seems like more trouble than it's worth rn